website-html-to-markdown/moveover/testing/05-test-frontmatter/no-cover-no-sidecar.md

```diff
@@ -0,0 +1,18 @@
+---
+order: 1
+title: Page with no sidecar and no cover
+category: testing
+layout:
+  cover: false
```

Comment:
Only question I have here is whether the property should be have the same name in the frontmatter. Probably doesn't matter, but thinking of bulk find/replace/search

Reply:
@jorytindall can you expand on this? are you suggesting to change this to `hasCover` or change `hasCover` to `cover`?

(the thinking behind the different naming is in the context of code, it's more common to use `has` or `is` as prefix for "logical/binary" variables, while for the frontmatter I favoured simplicity for the editor (easy to remember, and more or less all the frontmatter attributes for now have very simple (and memorable) names.

Reply:
Yes, that's what I was referring to, making the frontmatter property consistent with the code property (`hasCover` and `hasSidecar`). For context we mirror that property naming in Figma. Though it also makes sense for simplicity in the frontmatter.

Not necessarily advocating for one over the other, more just raising out of curiousity.

Reply:
Keep in mind that there will be some (rare) conditions in which, even if the `sidecar` is not explicitly set to `false` (so it's should be `true` by default, in reality the sidecar will be hidden anyway (eg. no headings of a certain level in the page); the `hasSidecar` encompasses more than just the frontmatter value.